.gitignore

```diff
@@ -1,6 +1,6 @@
 # Misc
 .DS_Store
-logs/
+logs1/
```

Comment:
Is this change intended?

Reply:
Perhaps this could be covered by a glob for all log type files?

Reply:
Not intended, and yes good idea, thanks!